### ISSUE TITLE
Validity check not working according to 2G rules

### ISSUE THREAD

**Issue by mejmo** [corona-warn-app/cwa-app-android#4829]
The app does not do verification for Austria correctly. They have completely different 2G rules, so in the app it shows the certificate is valid for Austria but it is not because last vaccination was more than half a year ago. Please fix, I consider this as quite critical because it can spoil travel plans.

**Comment by Ein-Tim** [comment]
You mean clicking on "Reisegültigkeit prüfen" and then selecting "Österreich" leads to this wrong result?

**Comment by mejmo** [comment]
Yes. For Austria is the vaccination of the second shot older than 6 months considered as not vaccinated at all. If I am wrong please correct me, it might be that I do not understand that feature.

**Comment by mejmo** [comment]
If it is only for entering the country then the title "Your certificate satisfy 2G rule" is wrong. Because it does not satisfy nor austrian nor german 2G rule.

**Comment by Ein-Tim** [comment]
Okay, as the app is intended for use in Germany the 2G(+) badges are shown according to German rules.

For Austria it's only relevant that the validity check for entering the country is correct
